### DISCUSSION TITLE
AppControl: Force Stop apps via accessibility service

### DISCUSSION THREAD

**Discussion by nightznero** [d4rken-org/sdmaid-se#677] [Ideas]
Helloz!
I would like to see Freeze Apps feature in SD  Maid SE,i missed that in Maid 1 and  would be awesome to have it in her continuation. I think its nice feature to save battery and eith it i would delete others app that doing that.
Best Regards, NIGHTZNERO.

**Comment by d4rken** [comment]
AppControl already supports that. Needs root or Shizuku.

**Comment by nightznero** [comment]
I was thinking it easier to change "memeory" in code(that is used to clean cache) to "force stop" but see there is only the way to have root,or have root. Bcs i see there no point to connect to PC with adb everytime i want to close all apps. This is even worse than haveing 3rd party apps to close apps directly from phone. Well,weird but if its not planned rip idea

**Comment by d4rken** [comment]
Freezing apps, i.e. disabling apps, is only possible with root or shizuku (shizuku requires setup once per reboot, not everytime).

Force stopping apps via ACS is not the same as disabling apps. I'm wary of such a feature, SD Maid is not a "task killer" :thinking:.

**Comment by nightznero** [comment]
I mean yeah,uncorrected said by me. Difnt mean to freez app but more correctly,task. I know and understand for what first reason You created a SD Maid. It didn't mean to be a task killer but cleaner from garbage leftovers. But as "Maid" in name she can also clean task. Then Maid would cover whole device. Buuuut i understand if the main goal of this app and why would you reject a suggestion
